src/util/api.js

```diff
@@ -65,6 +57,13 @@ const post = (path, body) => {
         return res.json();
       }
       return res.text();
+    })
+    .then(res => {
+      if (res.status >= 400) {
+        const e = res;
+        throw e;
+      }
+      return res;
```

Comment:
Is `res` an instance of Error here? I think it's a generic convention in JavaScript to only throw Error instances, even if the language allows throwing pretty much anything.

I guess this is just a convenient way to trigger the default error printing, right?

Reply:
@kpuputti I guess the problem is that the "res" is a JSON object. Using plain Error is a bit problematic since its constructor is quite picky compared to plain _throw_. So, it's either `throw data;` or `throw new MyCustomError(data);`

If I read this correctly, Axios uses plain Promise.reject (i.e. pretty much same approach as plain "throw"): https://github.com/axios/axios/blob/0d69a79c81a475f1cca6d83d824eed1e5b0b045d/lib/core/dispatchRequest.js#L77

Reply:
@kpuputti Good point! I didn't even think about this because it was possible to throw the data object directly. I will change this to Error instance,